dev/fattest.simplicity/test/componenttest/custom/junit/runner/CompoundFilterTests.java

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package componenttest.custom.junit.runner;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.junit.runner.Description;
+import org.junit.runner.RunWith;
+import org.junit.runner.manipulation.Filter;
+
+import componenttest.app.FATServlet;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import jakarta.servlet.annotation.WebServlet;
+
+/**
+ *
+ */
+public class CompoundFilterTests {
+    private static final Filter[] testFiltersToApply = new Filter[] {
+                                                                      new TestModeFilter(),
+                                                                      new TestNameFilter(),
+                                                                      new FeatureFilter(),
+                                                                      new SystemPropertyFilter(),
+                                                                      new JavaLevelFilter(),
+                                                                      new CheckpointSupportFilter()
+    };
+
+    @Mode(TestMode.FULL) //Should be superseeded by servlet or method annotation
```

Comment:
Throughout the file you are using the word as `superseed`, but that is the wrong spelling.  It should be `supersede`.